recipes/openjph/build.sh

```diff
@@ -0,0 +1,14 @@
+set -ex
+
+mkdir -p build
+pushd build
+
+# OJPH_BUILD_EXECUTABLES would require libtiff which creates a circular
+# dependency
+cmake ${CMAKE_ARGS}                          \
+    -DOJPH_BUILD_EXECUTABLES=ON              \
+    ..
+
+make -j ${CPU_COUNT}
```

Comment:
I recommend ninja everywhere, but up to you!

Reply:
so much muscle memory.